inicheck/changes.py

```diff
@@ -228,7 +221,7 @@ def apply_changes(self, ucfg, potentials, changes):
                 s_n = c[1][0]
                 i_n = c[1][1]
 
-            #print(s_o, i_o, s_n, i_n)
+            # print(s_o, i_o, s_n, i_n)
```

Comment:
Any reason to keep this around?

Reply:
No, just tried to keep changes minimal but I can remove